dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -19,188 +19,222 @@
+)
+@click.option(
+    "--pid-file", type=str, default="", help="File to write the process PID.",
+)
+@click.option(
+    "--resources",
+    type=str,
+    default="",
+    help="""Resources for task constraints like ``"GPU=2 MEM=10e9"``. Resources are
+    applied separately to each worker process (only relevant when starting multiple
+    worker processes with ``--nprocs``).""",
 )
 @click.option(
-    "--reconnect/--no-reconnect",
+    "--dashboard/--no-dashboard",
+    "dashboard",
     default=True,
-    help="Reconnect to scheduler if disconnected",
+    show_default=True,
+    required=False,
+    help="Launch the dashboard.",
 )
-@click.option("--pid-file", type=str, default="", help="File to write the process PID")
 @click.option(
-    "--local-directory", default=None, type=str, help="Directory to place worker files"
+    "--dashboard-address",
+    type=str,
+    default=":0",
+    show_default=True,
+    help="Relative address to serve the dashboard (if enabled).",
 )
 @click.option(
-    "--resources",
+    "--local-directory",
+    default=None,
     type=str,
-    default="",
-    help='Resources for task constraints like "GPU=2 MEM=10e9". '
-    "Resources are applied separately to each worker process "
-    "(only relevant when starting multiple worker processes with '--nprocs').",
+    help="""Path on local machine to store temporary files. Can be a string (like
+    ``"path/to/files"``) or ``None`` to fall back on the value of
+    ``dask.temporary-directory`` in the local Dask configuration, using the current
+    working directory if this is not set.""",
 )
 @click.option(
     "--scheduler-file",
     type=str,
     default="",
-    help="Filename to JSON encoded scheduler information. "
-    "Use with dask-scheduler --scheduler-file",
+    help="""Filename to JSON encoded scheduler information. To be used in conjunction
+    with the equivalent ``dask-scheduler`` option.""",
 )
 @click.option(
-    "--dashboard-prefix", type=str, default=None, help="Prefix for the Dashboard"
+    "--interface",
+    type=str,
+    default=None,
+    help="""External interface used to connect to the scheduler. Usually an ethernet
+    interface is used for connection, and not an InfiniBand interface (if one is
+    available). Can be a string (like ``"eth0"`` for NVLink or ``"ib0"`` for
+    InfiniBand) or ``None`` to fall back on the default interface.""",
 )
 @click.option(
     "--preload",
     type=str,
     multiple=True,
     is_eager=True,
-    help="Module that should be loaded by each worker process "
-    'like "foo.bar" or "/path/to/foo.py"',
+    help="""Module that should be loaded by each worker process like ``"foo.bar"`` or
+    ``"/path/to/foo.py"``.""",
 )
-@click.argument(
-    "preload_argv", nargs=-1, type=click.UNPROCESSED, callback=validate_preload_argv
+@click.option(
+    "--dashboard-prefix",
+    type=str,
+    default=None,
+    help="""Prefix for the dashboard. Can be a string (like ...) or ``None`` for no
```

Comment:
Do we know what dashboard prefix input should look like?

Reply:
I'm not sure, maybe @jacobtomlinson can help with this one.